docker/test/integration/environment.py

```diff
@@ -30,7 +34,7 @@ def before_scenario(context, scenario):
         return
 
     logging.info("Integration test setup at {time:%H:%M:%S.%f}".format(time=datetime.datetime.now()))
-    context.test = MiNiFi_integration_test(context.image_store)
+    context.test = MiNiFi_integration_test(context.test_id, context.image_store, context.directory_bindings, context.kind)
```

Comment:
It may be nicer to pass the whole `context` object through all the way from here `-> MiNiFi_integration_test -> DockerTestCluster -> SingleNodeDockerCluster` and let classes take what they need from it along the way.
```suggestion
    context.test = MiNiFi_integration_test(context)
```